src/quo/components/drawers/bottom_actions/style.cljs

```diff
@@ -21,12 +21,13 @@
                        (colors/theme-colors colors/white colors/neutral-95 theme))})
 
 (defn buttons-container
-  [actions]
-  {:flex-direction     (if (= actions :two-vertical-actions) :column :row)
-   :justify-content    :space-around
-   :padding-vertical   12
-   :gap                12
-   :padding-horizontal 20})
+  [actions buttons-container-style]
```

Comment:
nit: can just be `container-style` here

Reply:
I also feel `container-style` is more apt.

Reply:
it cannot be since there is already `container-style`